ironfish/src/node.ts

```diff
@@ -156,6 +164,11 @@ export class IronfishNode {
       await internal.load()
     }
 
+    if (!hosts) {
+      hosts = new HostsStore(files, dataDir)
+      await hosts.load()
+    }
+
```

Comment:
Again, it looks like this store is going to be node level instead of being networking related. What's the larger vision around this hosts list store? Will block chain use it? Will mem pool use it for something?